### PR TITLE
Add partners/kubernetes page

### PR DESCRIPTION
**Pull request by solazio**
## Done
- Add `/partners/kubernetes` page

## QA
- Run the website locally using `./run`
- Go to [/partners/kubernetes](http://0.0.0.0:8002/partners/kubernetes) and compare to [design](https://github.com/canonical-web-and-design/web-squad/issues/1763)

## Issue
Fixes https://github.com/canonical-web-and-design/web-squad/issues/1763

### CODE REVIEW

File: templates/partners/kubernetes.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        </div>
+        <div class="col-6 u-align--center u-hide--small">
+          <img src="https://assets.ubuntu.com/v1/0038a560-charmed-kubernetes-partners-cloud.svg" alt="charmed kubernetes image">
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="p-strip is-deep u-extra-space">
+    <div class="row u-sv3">
+      <p class="p-heading--two">Technology and product partners</p>
```
Comment (sowasred2012) [inline templates/partners/kubernetes.html]: This should be a `<h2>` element, at the moment the page heirarchy jumps from the `<h1>` header on line 8, straight to the `<h3>` headers on lines 24, 31 and 41.

Screenshot 2019-10-07 at 17 03 11

File: templates/partners/kubernetes.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      </ul>
+    </div>
+  </section>
+  <section class="p-strip--mid-light is-deep">
+    <div class="row p-divider">
+      <div class="col-6 p-divider__block">
+        <div class="p-media-object">
+          <img src="https://assets.ubuntu.com/v1/2cdff5cc-magnifier-find-a-partner-footer.svg" class="p-media-object__image">
+          <div class="p-media-object__details">
+            <h1 class="p-media-object__title">Find a partner</h1>
```
Comment (sowasred2012) [inline templates/partners/kubernetes.html]: This should also be a `<h2>`, semantically it doesn't belong on the same level as the main title in the page heirachy.

It looks smaller than in the design, too. Maybe `.p-heading--two` would work?

Screenshot 2019-10-07 at 17 08 05

File: templates/partners/kubernetes.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            <p class="p-media-object__content u-sv3">Looking for a supplier with Ubuntu expertise? Select from the complete list of our partners.</p>
+            <a href="/partners/find-a-partner">Find a partner&nbsp;&rsaquo;</a>
+          </div>
+        </div>
+      </div>
+      <div class="col-6 p-divider__block">
+        <div class="p-media-object">
+          <img src="https://assets.ubuntu.com/v1/292d1694-become-a-partner-footer.svg" class="p-media-object__image">
+          <div class="p-media-object__details">
+            <h1 class="p-media-object__title">Become a partner</h1>
```
Comment (sowasred2012) [inline templates/partners/kubernetes.html]: This should also be a `<h2>`, semantically it doesn't belong on the same level as the main title in the page heirachy.

It looks smaller than in the design, too. Maybe `.p-heading--two` would work?

File: templates/partners/kubernetes.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        <h3 class="p-card__title" style="padding-top: 3rem;">JFrog Artifactory</h3>
+        <p class="p-card__content u-sv3">JFrog Artifactory is the enterprise-ready Kubernetes registry fully integrated with both Rancher and the Charmed Distribution of Kubernetes. Long known to developers, Artifactory provides the governance, audit capability and artifact management. Canonical partners with JFrog to integrate CDK with a single point and click installation using conjure-up.</p>
+        <!-- Add link url -->
+        <a href="" class="p-button--positive">Watch the JFrog Artifactory webinar</a>
+      </div>
+    </div>
+  </section>
+  <section class="p-strip--light is-slanted--top-right is-deep">
+    <div class="row">
+      <p class="p-heading--two">Partner benefits:</p>
```
Comment (sowasred2012) [inline templates/partners/kubernetes.html]: This should be a `<h2>` element too.

File: templates/partners/kubernetes.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    <div class="row">
+      <div class="p-card--no-border col-4">
+        <img class="p-card__thumbnail-large" src="https://assets.ubuntu.com/v1/8f878ba0-rancher-logo-stacked-color.svg" alt="rancher image">
+        <h3 class="p-card__title" style="padding-top: 3rem;">Rancher Labs</h3>
+        <p class="p-card__content u-sv3">Rancher provides a turn-key application delivery platform, built on Ubuntu and the Charmed Distribution of Kubernetes (CDK). Maximise developer velocity, integrate CI/CD, and ease the path from development into production with enterprise-calibre container management. Canonical delivers a cloud-native platform in partnership with Rancher Labs.</p>
+        <a href="" class="p-button--positive">Watch the Introduction webinar</a>
+      </div>
+      <div class="p-card--no-border col-4">
+        <!-- Update image -->
+        <img class="p-card__thumbnail-large" src="https://assets.ubuntu.com/v1/8f878ba0-rancher-logo-stacked-color.svg" alt=" image">
```
Comment (sowasred2012) [inline templates/partners/kubernetes.html]: Is this a placeholder image? It doesn't match the design (I know there's some lorem ipsum, so wasn't sure). Also, the images are left aligned, but center aligned in the design.

**Comment by waynecrosby** [comment]
1) Move the header illustration up so that it is centered vertically
2) Reduce the gap between the header and the copy 'Technology and..' by half
3) Center the JFrog logo horizontally

**Comment by waynecrosby** [comment]
1) Move the header illustration up so that it is centered vertically
2) The gap between the headline and sub copy is too big
Other amends look good

**Pull request merged**